protected/humhub/modules/like/controllers/LikeController.php

```diff
@@ -78,6 +83,9 @@ public function actionLike()
      */
     public function actionUnlike()
     {
+        if (!$this->module->canLike($this->parentContent)) {
```

Comment:
Maybe unlike should always be possible, a user should always be able to revert such actions? But I think this would only be an edge case.

Reply:
> Maybe unlike should always be possible, a user should always be able to revert such actions? But I think this would only be an edge case.

All right, I will remove this check for `actionUnlike`. Thank you for this note